src/main/java/ua/com/danit/service/CommentService.java

```diff
@@ -0,0 +1,55 @@
+package ua.com.danit.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ua.com.danit.entity.Comment;
+import ua.com.danit.entity.Post;
+import ua.com.danit.entity.User;
+import ua.com.danit.repository.CommentRepository;
+
+@Service
+public class CommentService {
+  private CommentRepository commentRepository;
+  private UserService userService;
+
+  @Autowired
+  public CommentService(CommentRepository commentRepository, UserService userService) {
+    this.commentRepository = commentRepository;
+    this.userService = userService;
+  }
+
+  public Comment findCommentById(Long commentId) {
+    return commentRepository.findCommentById(commentId);
+  }
+
+  public void addComment(Long postId, String text) {
+    Post post = new Post(); // TODO Get post by ID
```

Comment:
add search by id from PostRepository

Reply:
PostRepository is not yet available

Reply:
feel free to create neccessary for you data

Reply:
@ShevchukMarina has already created, but has not yet made a push

Reply:
it's not a problem, repository is empty interface, it will be easy to merge it